config/playground/demo/observability/otel_demo/nginx.yaml

```diff
@@ -0,0 +1,166 @@
+# Copyright The OpenTelemetry Authors
+# SPDX-License-Identifier: Apache-2.0
+---
+# Nginx Deployment
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: nginx
+  namespace: otel-demo
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: nginx
+  template:
+    metadata:
+      labels:
+        app: nginx
+    spec:
+      containers:
+        - name: nginx
+          image: nginx:latest
+          ports:
+            - containerPort: 90
+          volumeMounts:
+            - name: config-volume
+              mountPath: /etc/nginx/conf.d/default.conf
+              subPath: default.conf
+            - name: log-storage
+              mountPath: /var/log/nginx/
+        - name: fluent-bit
+          image: fluent/fluent-bit
+          env:
+            - name: OPENSEARCH_HOST
+              value: "opensearch-cluster-leader.default.svc"
+            - name: OPENSEARCH_USER
+              value: "{USR}"
+            - name: OPENSEARCH_PWD
+              value: "{PSWD}"
+          volumeMounts:
+            - name: log-storage
+              mountPath: /var/log/
+            - name: fluent-bit-config
+              mountPath: /fluent-bit/etc/
+      restartPolicy: Always
+      volumes:
+        - name: config-volume
+          configMap:
+            name: nginx-config
+            items:
+              - key: default.conf
+                path: default.conf
+        - name: opentelemetry-volume
+          configMap:
+            name: nginx-config
+            items:
+              - key: opentelemetry_module.conf
+                path: opentelemetry_module.conf
+        - name: log-storage
+          emptyDir: {}
+        - name: fluent-bit-config
+          configMap:
+            name: fluent-bit-sidecar-config
+---
+# Nginx Config-map
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: nginx-config
+  namespace: otel-demo
+data:
+  default.conf: |
+    server {
+      listen 90;
+      access_log /var/log/nginx/frontend.access.log;
+      error_log /var/log/nginx/frontend.error.log;
+      location / {
+         proxy_pass http://opentelemetry-demo-frontend:8080/;
+         proxy_set_header Host $host;
+         proxy_set_header X-Real-IP $remote_addr;
+         proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+         proxy_set_header X-Forwarded-Proto $scheme;
+       }
+    }
+---
+# fluent-bit Config-map
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: fluent-bit-sidecar-config
+  namespace: otel-demo
+  labels:
+    app: nginx
+data:
+  fluent-bit.conf: |
+    [SERVICE]
+        Flush         1
+        Log_Level     info
+        Daemon        off
+        Parsers_File  parsers.conf
+        Log_Level     debug
+    @INCLUDE input-nginx.conf
+    @INCLUDE output-opensearch.conf
+  input-nginx.conf: |
+    [INPUT]
+        Name              tail
+        Tag               nginx.*
```

Comment:
frontend.access for tag. this is workable version for yesterday. I forgot to copy the yesterday's version